chart/openfaas/README.md

```diff
@@ -189,6 +189,7 @@ Additional OpenFaaS options in `values.yaml`.
 | `gateway.scaleFromZero` | Enables an intercepting proxy which will scale any function from 0 replicas to the desired amount | `true` |
 | `queueWorker.replicas` | Replicas of the queue-worker, pick more than `1` for HA | `1` |
 | `queueWorker.ackWait` | Max duration of any async task/request | `60s` |
+| `nats.enableMonitoring` | Enable the NATS monitoring endpoints  | `false` |
```

Comment:
Let's mention what port that opens up?

Reply:
I almost included that initially, but I wasn't sure. I will add it now